contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,1186 @@
+        createBid_l34(_bidTokenId);
+    }
+
+    /**
+     * @notice Settles a completed auction for `_tokenId`, if it exists and is
+     * not yet settled.
+     * Returns early (does not modify state) if
+     *   - there is no initialized auction for the project
+     *   - there is an auction that has already been settled for the project
+     *   - there is an auction for a different token ID on the project
+     *     (likely due to a front-run)
+     * This function reverts if the auction for `_tokenId` exists, but has not
+     * yet ended.
+     * @param _tokenId Token ID to settle auction for.
+     */
+    function settleAuction(uint256 _tokenId) public nonReentrant {
+        uint256 _projectId = _tokenId / ONE_MILLION;
+        ProjectConfig storage _projectConfig = projectConfig[_projectId];
+        Auction storage _auction = _projectConfig.activeAuction;
+        // CHECKS
+        // @dev this check is not strictly necessary, but is included for
+        // clear error messaging
+        require(_auction.initialized, "Auction not initialized");
+        if (_auction.settled || (_auction.tokenId != _tokenId)) {
+            // auction already settled or is for a different token ID, so
+            // return early and do not modify state
+            return;
+        }
+        require(block.timestamp > _auction.endTime, "Auction not yet ended");
+        // EFFECTS
+        _auction.settled = true;
+        // INTERACTIONS
+        // send token to the winning bidder
+        IERC721(genArt721CoreAddress).transferFrom(
+            address(this),
+            _auction.currentBidder,
+            _tokenId
+        );
+        // distribute revenues from auction
+        splitRevenuesETH(_projectId, _auction.currentBid, genArt721CoreAddress);
+
+        emit AuctionSettled(
+            _tokenId,
+            _auction.currentBidder,
+            _auction.currentBid
+        );
+    }
+
+    /**
+     * @notice Enters a bid for token `_tokenId`.
+     * If an auction for token `_tokenId` does not exist, an auction will be
+     * initialized as long as any existing auction for the project has been
+     * settled.
+     * In order to successfully place the bid, the token bid must be:
+     * - greater than or equal to a project's minimum bid price if a new
+     *   auction is initialized
+     * - sufficiently greater than the current highest bid, according to the
+     *   minter's bid increment percentage `minterMinBidIncrementPercentage`,
+     *   if an auction for the token already exists
+     * If the bid is unsuccessful, the transaction will revert.
+     * If the bid is successful, but outbid by another bid before the auction
+     * ends, the funds will be noncustodially returned to the bidder's address,
+     * `msg.sender`. A fallback method of sending funds back to the bidder via
+     * WETH is used if the bidder address is not accepting ETH (preventing
+     * denial of service attacks) within a 30_000 gas limit.
```

Comment:
Any reason we would want to consider to make this gas limit configurable for future-proofing purposes?

Reply:
I think this is reasonable to be an admin-configurable value (if even for improved cross-chain compatibility).
The value is easily packed with other minter-configuration values also accessed during a new bid, so the gas impacts are minimal.
I've updated in commit [98394da](https://github.com/ArtBlocks/artblocks-contracts/pull/504/commits/98394da2c3fdce41e609d2a677a683ca85081e82) with a solution that emits an event and includes the value in our `minterConfigurationDetails` view function.
I would recommend that we don't bother indexing this value in our subgraph, just because it would be seldomly changed. Thoughts?

Reply:
Fantastic – that rationale SGTM, I don't think we need to index this info right now.

One minor follow up comment on min value being checked, but can follow up on that in the separate comment thread where I flagged it – resolving!